layouts/shortcodes/sitemap.html

```diff
@@ -0,0 +1,25 @@
+{{ $envURL := ( index .Site.Params.envURL (os.Getenv "BRANCH") ) }}
+{{ $envURL := cond (eq $envURL nil) .Site.Params.envURL.fed $envURL }}
+
+<div class="sitemap">
+  <h2>Main Navigation</h2>
+  <ul class="main-nav usa-list">
+      {{ $translation := ( index $.Site.Data.translations .Site.Language.Lang ) }}
+      <li><a hreflang="{{ $translation.Language.Lang }}" href="{{ .Site.Home.RelPermalink }}">Home</a></li>
+      {{ range where .Site.RegularPages "Section" "!=" .Site.Params.register_path }}
+        {{ if ne .Title "Sitemap" }}
+          <li><a hreflang="{{ .Language.Lang }}" href="{{ .Permalink }}">{{ .Title }}</a></li>
+        {{ end }}
+      {{ end }}
+  </ul> 
+
+  <h2>Register to Vote - by State and Territory</h2>
```

Comment:
Replace "Register to Vote - by State and Territory" with a variable and add it to the front matter in the md translations.